test/common/upstream/upstream_impl_test.cc

```diff
@@ -251,7 +256,33 @@ TEST(StaticClusterImplTest, UnsupportedFeature) {
   )EOF";
 
   Json::StringLoader config(json);
-  EXPECT_THROW(StaticClusterImpl(config, stats, ssl_context_manager), EnvoyException);
+  EXPECT_THROW(StaticClusterImpl(config, runtime, stats, ssl_context_manager), EnvoyException);
+}
+
+TEST(ResourceManagerImplTest, RuntimeResourceManager) {
+  NiceMock<Runtime::MockLoader> runtime;
+  ResourceManagerImpl resource_manager(
+      runtime, "circuit_breakers.runtime_resource_manager_test.default.", 0, 0, 0, 0);
+
+  EXPECT_CALL(
```

Comment:
this is not a useful test. Same for all of the ones below. You need to actually return a non-default value in your expectation, and then verify that is what is returned.